### PR TITLE
Update FDR to version 4

### PR DESCRIPTION
**Pull request by imc0**
The app changed its name from FDR3 to FDR4 so needs a version change in the pkg recipe
and a pre-install script to clean up the old app folder when the package is upgraded.

### CODE REVIEW

**Comment by imc0** [comment]
I'm having a bit of trouble with this.  I think pkgutil's caches need to be cleaned for this recipe or it doesn't build correctly.

**Comment by fuzzylogiq** [comment]
Taking it that you mean autopkg rather than pkgutil, can you not just use `purge_destination` when using `Unarchiver`? https://github.com/autopkg/autopkg/wiki/Processor-Unarchiver

**Comment by imc0** [comment]
Yes, I believe you are right.  Problem was that after unpacking FDR4.app the packager was picking up the old FDR3.app directory from the cache.  So that should be fixed now.
(Other issue I was having must have been local as a reboot fixed it.)

File: FDR/FDR.munki.recipe [hunk 1]
```diff
@@ -48,6 +48,16 @@
       <string>Software Development</string>
       <key>description</key>
       <string>FDR analyses programs written in CSP_M, which combines the operators of Hoare's CSP with a functional programming language. It also has support for analysing timed systems, via tock CSP.</string>
+      <key>preinstall_script</key>
+      <string>#!/bin/sh
+# If upgrading from FDR3 don't leave the old FDR3.app folder lying around
+if pkgutil --files uk.ac.ox.orchard.pkg.FDR 2>/dev/null | grep -q Applications/FDR3\\.app/
```
Comment (fuzzylogiq) [inline FDR/FDR.munki.recipe]: similarly to the Oxygen recipe, if these references to the pkg id could be made an Input variable to the pkg recipe that would be great.

**Comment by imc0** [comment]
OK how is that?

**Comment by fuzzylogiq** [comment]
LGTM

**Pull request merged**